task_templates/3_pipelines/2_python3_sklearn_regression/create_pipeline.py

```diff
@@ -92,10 +92,21 @@ def to_string(x):
 # For text, we:
 # 1. Impute missing values with the string "missing"
 # 2. Tfidf encode the text, using 1-grams and 2-grams.
+
+# Sklearn's TfidfVectorizer can only handle one column of text at a time,
```

Comment:
this comment is exactly the same as below.  please remove it.  I'm assuming that this works with the current dataset?

Reply:
done